app/components/input-group/component.js

```diff
@@ -0,0 +1,20 @@
+import Component from '@ember/component';
+import {computed} from '@ember/object';
+
+export default Component.extend({
+  init() {
+    this._super();
+    if (this.defaultValue) {
```

Comment:
so that I believe wouldn't be needed in Ember - without knowing the details of what you are doing. Can just optionally pass `inputValue` as an argument; or use `computed.oneWay` to set it initially and allow override etc.

The idea is that, in general, we can do without using the `init` / `didInsertElement` etc. hooks, and all with computed properties

Reply:
rest is all good, approving if you want to check that part later